packages/compiler/src/core/linter.ts

```diff
@@ -21,9 +21,6 @@ import {
 export interface Linter {
   extendRuleSet(ruleSet: LinterRuleSet): Promise<readonly Diagnostic[]>;
   lint(): readonly Diagnostic[];
-
-  /** @internal */
-  getRuleUrl(ruleId: string): string | undefined;
```

Comment:
short lived internal api :D turns out needing the program to get the url is very limiting so its easier if we allow each diagnostic to include a url

Reply:
Since this is exported, shouldn't we deprecate it now and remove later?

Reply:
I just added it this morning, also I don't think anything marked with `@internal` should have to go through this process. Anything marked with `@internal` won't have its type generated